app/templates/components/events/view/overview/event-tickets.hbs

```diff
@@ -15,7 +15,7 @@
       </tr>
       <tr>
         <td>{{t 'Sales (amount)'}}</td>
-        <td>{{currency-symbol this.data.event.paymentCurrency this.sales}} {{format-money this.sales}}</td>
+        <td><CurrencyAmount @currency={{currency-symbol this.data.event.paymentCurrency}} @amount={{format-money this.sales}}/></td>
```

Comment:
What's the point of component if you are doing calculations here. currency-symbol should not be called here. It should be called inside the component

Reply:
ok but for amount I have to do that it here only bcz for diff it is diff